src/components/map-marker/map-marker.e2e.ts

```diff
@@ -0,0 +1,103 @@
+import { E2EElement, E2EPage, newE2EPage } from "@stencil/core/testing";
+
+describe("gx-map-marker", () => {
+  let parentElement: E2EElement;
+  let element: E2EElement;
+  let page: E2EPage;
+
+  beforeEach(async () => {
+    page = await newE2EPage();
+  });
+
+  it("should work w/o attributes", async () => {
+    await page.setContent("<gx-map><gx-map-marker></gx-map-marker></gx-map>");
+    await page.waitForChanges();
+    parentElement = await page.find("gx-map");
+    element = await parentElement.find("gx-map-marker");
+    ///////////////////////////////////////
+    const innerContent = await parentElement.find(".leaflet-marker-icon");
+    expect(!!innerContent).toEqual(true);
+    expect(await element.getProperty("coords")).toEqual("0, 0");
+    expect(await element.getProperty("iconSrc")).toEqual(
+      "https://unpkg.com/leaflet@1.3.4/dist/images/marker-icon.png"
+    );
+    expect(await element.getProperty("iconSize")).toEqual("25, 41");
+    expect(await element.getProperty("iconAnchor")).toEqual("12.5, 41");
+    expect(await element.getProperty("tooltipAnchor")).toEqual("0, -30");
+    expect(await element.getProperty("tooltipCaption")).toEqual(undefined);
+  });
+  it("should set a given coords (but w/ default properties)", async () => {
```

Comment:
Please leave an empty line between each test to improve code readabilty

Reply:
Ready.

Reply:
Ready.

Reply:
Please verify the other tests too. There are still some lines needed to separate the other tests.